io/zenoh-links/zenoh-link-shm/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2023 ZettaScale Technology
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+# which is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+#
+# Contributors:
+#   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+#
+[package]
+rust-version = { workspace = true }
+name = "zenoh-link-shm"
+version = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+license = { workspace = true }
+categories = { workspace = true }
+description = "Internal crate for zenoh."
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+transport_shm = []
+
+[dependencies]
+async-std = { workspace = true }
+async-trait = { workspace = true }
+async-io = ">= 1.13.0"
+log = { workspace = true }
+rand = { workspace = true }
+socket2 = { workspace = true }
+zenoh-buffers = { workspace = true }
+zenoh-collections = { workspace = true }
+zenoh-core = { workspace = true }
+zenoh-config = { workspace = true }
+zenoh-link-commons = { workspace = true }
+zenoh-protocol = { workspace = true }
```

Comment:
I don't think `zenoh-protocol` should be a dependency of `zenoh-link-shm`.

Reply:
Locator and EndPoint are defined in zenoh-protocol

Reply:
But I removed other unnecessary stuff here, thank you